randyhub/src/common/Chaos/styles.module.scss

```diff
@@ -1,22 +1,13 @@
+@import '../../styles/colors';
+@import '../../styles/fonts';
+
 .chaos {
+  @include body-text;
+
   display: flex;
   flex-direction: column;
-  font-family: adelle-sans, sans-serif;
-  font-weight: 100;
-  font-size: 1.2em;
-  color: #ff5349;
-
-  a {
-    text-decoration: none !important;
-    color: #ff5349;
-    margin: 0 5px;
-    transition: 0.5s;
-
-    &:hover {
-      font-weight: 100;
-      color: #ffffff;
-    }
-  }
+  font-size: 1em;
```

Comment:
Does a default font-size of 1em do anything? Isn't that like saying "default font size?"

Reply:
Good question - I think it follows browser's native specs, it's worked a lot more smoothly for me for extremely high DPI screens and/or screens with weird resolutions and base font sizes.